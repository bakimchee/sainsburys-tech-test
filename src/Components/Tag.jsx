import styled from 'styled-components';
import { theme } from '../Theme';

const Tag = styled.span`
  background-color: ${theme.color.palette.blue};
  border-radius: ${theme.size.spacing.default};
  padding: 0.25rem 0.5rem;
`;

export default Tag;
