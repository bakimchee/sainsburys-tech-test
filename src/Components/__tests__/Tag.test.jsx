/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Tag from '../Tag';

test('Tag', () => {
  const component = renderer.create(<Tag>3</Tag>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
