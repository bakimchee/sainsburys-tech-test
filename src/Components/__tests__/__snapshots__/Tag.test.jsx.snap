// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag 1`] = `
<span
  className="Tag-sc-14cv7o2-0 lbMyPj"
>
  Add to basket
</span>
`;
